control-plane/agents/src/bin/core/tests/rebuild/mod.rs

```diff
@@ -12,8 +12,9 @@ use stor_port::types::v0::{
             ReplicaState, VolumePolicy, VolumeStatus,
         },
     },
-    transport::VolumeId,
+    transport::{Filter, GetRebuildRecord, NexusId, NexusStatus, VolumeId},
 };
+const REBUILD_WAIT_TIME: u64 = 40;
```

Comment:
wait time in what?

Reply:
This is the wait time we have for nexus to be Online again. its reduced to 10s now.